packages/block-library/src/columns/style.scss

```diff
@@ -26,11 +29,9 @@
 
 	@include break-small() {
 
-		// Beyond mobile, allow 2 columns.
-		flex-basis: calc(50% - #{$grid-size-large});
```

Comment:
I'm assuming this wasn't strictly necessary, also by the fact that it (wrongly) assumes a Columns block would only ever have 2 columns. In some testing, the default width distribution of a columns without widths assigned (`flex-basis: auto;`) appears to be equal distribution.

Reply:
> I'm assuming this wasn't strictly necessary, also by the fact that it (wrongly) assumes a Columns block would only ever have 2 columns. In some testing, the default width distribution of a columns without widths assigned (`flex-basis: auto;`) appears to be equal distribution.

Actually, in some final testing, it does appear to make a difference. I'm not sure if it's _intended_ that mid-range viewports show as two-column, but that's how it behaves in master, and I'd rather leave it as it was.

![image](https://user-images.githubusercontent.com/1779930/57557094-d631be80-7346-11e9-86d5-26f375edea2c.png)

(Note: This applies only to Columns blocks where widths are not explicitly assigned)

Reply:
Your assumptions sound correct here @aduth 
The original design had 2 columns always showing at the medium breakpoint. You may not have noticed it cause It didn't actually work for the first couple versions of the editor. Got fixed here https://github.com/WordPress/gutenberg/pull/12199

I don't think that two column breakpoint would be missed too much if removing it would help what you're doing here.